src/chart/AccessibilityManager.ts

```diff
@@ -0,0 +1,94 @@
+import type { LayoutType, TickItem } from '../util/types';
+
+interface ContainerOffset {
+  top: number;
+  left: number;
+}
+
+interface InitiableOptions {
+  coordinateList: TickItem[];
+  mouseHandlerCallback: (e: any) => void;
+  container: HTMLElement;
+  layout: LayoutType;
+  offset: ContainerOffset;
+}
+
+export class AccessibilityManager {
+  private activeIndex = 0;
+
+  private coordinateList: InitiableOptions['coordinateList'] = [];
+
+  private mouseHandlerCallback: InitiableOptions['mouseHandlerCallback'];
+
+  private container: InitiableOptions['container'];
+
+  private layout: InitiableOptions['layout'] = 'horizontal';
+
+  private offset: InitiableOptions['offset'];
+
+  public init({ coordinateList = [], container, layout, offset, mouseHandlerCallback }: InitiableOptions) {
```

Comment:
nit: could this not be an actual constructor or is there a reason we need to create an instance before we add these properties?